### PR TITLE
Katib 0.10 Release Blog Post

### PR DESCRIPTION
**Pull request by andreyvelich**
I added Blog Post: "Kubeflow Katib: Scalable, Portable and Cloud Native System for AutoML".
Name is under discussion, if you have any other options please let me know.

/cc @jbottum @gaocegege @johnugeorge @castrojo @amsaha

### CODE REVIEW

**Comment by andreyvelich** [comment]
@kramachandran @theadactyl Can you help with this Blog Post review, please ?

**Comment by theadactyl** [comment]
Hi! Thanks for tagging me. Can we get an LGTM from a subject matter expert aside from blog author? I think there are some copy edits needed here as well, which I'd love help with. Otherwise looks good to me -- happy to approve when those are taken care of.

**Comment by andreyvelich** [comment]
> Hi! Thanks for tagging me. Can we get an LGTM from a subject matter expert aside from blog author? I think there are some copy edits needed here as well, which I'd love help with. Otherwise looks good to me -- happy to approve when those are taken care of.

Thank you @theadactyl!
Please let me know which parts can be modified in this Blog from your point of view.

**Comment by andreyvelich** [comment]
I removed WIP status.
/hold for the review.

**Review by gaocegege — COMMENTED** [review]
/lgtm

Thanks for your contribution! :tada: :+1:

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:43]: Let's get rid of the "moreover" here

Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:43]: Let's get rid of the "moreover"

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:76]: Can you fix the ordered list?

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:94]: Revise sentence to start with "Katib version 0.10" or rephrase to "The lates Katib version, 0.10, implements..."

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:101]: "Here are the motivations behind this feature"

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:103]: resources-> resource

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:108]: Can we standardize to either "Trial Template" or "Trial template" consistently through the post?

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:143]: Again, can we fix the ordered list?

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:212]: simplify to "Katib 0.10"

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:213]: Since this is the first mention of HP tuning, can we rephrase to "hyper-parameter (HP) tuning"?

File: _posts/2021-03-10-katib-0.10.md
Comment (theadactyl) [inline _posts/2021-03-10-katib-0.10.md:301]: let's cut "furthermore"

**Review by theadactyl — CHANGES_REQUESTED** [review]
Awesome post. Thanks so much for contributing this. Once you fix the copy edits (commented), I will approve ASAP.

File: _posts/2021-03-10-katib-0.10.md
Comment (andreyvelich) [reply _posts/2021-03-10-katib-0.10.md:213]: I think the first mention of HP was here: https://github.com/kubeflow/blog/blame/32ae07c1db42d680d1b843f0c414143da18c351c/_posts/2021-03-10-katib-0.10.md#L30.

**Comment by andreyvelich** [comment]
Thank you so much for the review @theadactyl!
I addressed your comments.

**Comment by jbottum** [comment]
@andreyvelich very nice effort.    I have 3 comments.

(this sentence might need some commas or broken in to two sentences) - Since Katib can execute various Kubernetes resources, users are able to run Experiments which can optimize not only ML models but also any software, code or program to make it more efficient with optimization algorithms provided by Katib.


(should "perform" be "provide a"?) We are continually working on the new Katib UI to perform better User Experience and native integration with Kubeflow central dashboard.

(this sentence probably needs to be broken into two or more sentences...and I struggle with best and worse in the same sentence) : Currently, Katib supports the Median Stopping Rule which stops a pending Trial X at step S if the Trial's best objective value by step S is worse than the median value of the running averages of all completed Trials' objectives reported up to step S.     Suggestion....   Currently, Katib supports the Median Stopping Rule.   The Medium Stopping rule stops if the Trial's best objective is lower than the median value of all Trials' objectives completed to that point.

**Comment by andreyvelich** [comment]
Thank you for the comments @jbottum! What do you think ?

> (this sentence might need some commas or broken in to two sentences) - Since Katib can execute various Kubernetes resources, users are able to run Experiments which can optimize not only ML models but also any software, code or program to make it more efficient with optimization algorithms provided by Katib.


Since Katib can execute various Kubernetes resources, users are able to run not only ML models optimization Experiments. They can also enhance any software, code or program to make it more efficient with optimization algorithms provided by Katib.

> (should "perform" be "provide a"?) We are continually working on the new Katib UI to perform better User Experience and native integration with Kubeflow central dashboard.

Make sense, changed.

> (this sentence probably needs to be broken into two or more sentences...and I struggle with best and worse in the same sentence) : Currently, Katib supports the Median Stopping Rule which stops a pending Trial X at step S if the Trial's best objective value by step S is worse than the median value of the running averages of all completed Trials' objectives reported up to step S. Suggestion.... Currently, Katib supports the Median Stopping Rule. The Medium Stopping rule stops if the Trial's best objective is lower than the median value of all Trials' objectives completed to that point.


Thanks! What do you think about this "Currently, Katib supports the Median Stopping Rule. The Medium Stopping rule stops a running Trial if the Trial's best objective value is lower than the median value of all succeeded Trials' objectives completed to that point."

**Review by theadactyl — APPROVED** [review]
LGTM. Great work!

**Comment by theadactyl** [comment]
LMK when all stakeholders are ready for merge.

**Comment by andreyvelich** [comment]
@gaocegege @johnugeorge If you are fine with the changes, please can you give your `lgtm` and we can move forward?

**Comment by andreyvelich** [comment]
Thanks everyone for the help on this blog!
/hold cancel

**Comment by theadactyl** [comment]
Congrats!

**Pull request merged**
